.github/workflows/test-workflow.yml

```
@@ -20,6 +20,14 @@ jobs:
       with:
         node-version: "${{ steps.nvm.outputs.NVMRC }}"
 
+    - name: Cache Dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ steps.nvm.outputs.NVMRC }}-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
```

@MattIPv4 Something like this? https://github.com/cdnjs/api-server/pull/24#discussion_r444174954

```suggestion
        restore-keys: |
          ${{ runner.os }}-node-${{ steps.nvm.outputs.NVMRC }}-
          ${{ runner.os }}-node-
```

---

Yeah, that's what the docs suggest should work.